concordium-sdk/src/main/java/com/concordium/sdk/responses/blocksummary/updates/keys/Authorization.java

```diff
@@ -0,0 +1,34 @@
+package com.concordium.sdk.responses.blocksummary.updates.keys;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.util.List;
+
+/**
+ * Level 2 key authorizations.
+ */
+@Getter
+@ToString
+@EqualsAndHashCode
+public final class Authorization {
+    /**
+     * The threshold
+     */
+    private final int threshold;
```

Comment:
Yes. Bytes are signed in java... So it could lead to confusion above 128 for the consumer. Thus I would prefer this was just an `int`, or even a `short` (signed I16) for having less code to maintain. 

We can do the other thing though.